src/top_model/main.cpp

```diff
@@ -146,19 +146,19 @@ std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> NETWORK = std::make_s
 /************************/
 cadmium::dynamic::modeling::Ports iports_ABPSimulator = {typeid(inp_control)};
 cadmium::dynamic::modeling::Ports oports_ABPSimulator = {typeid(outp_ack),typeid(outp_pack)};
-cadmium::dynamic::modeling::Models submodels_ABPSimulator = {sender1, receiver1,NETWORK};
+cadmium::dynamic::modeling::Models submodels_ABPSimulator = {sender1, receiver1, NETWORK};
 cadmium::dynamic::modeling::EICs eics_ABPSimulator = {
-  cadmium::dynamic::translate::make_EIC<inp_control, Sender_defs::controlIn>("sender1")
+  cadmium::dynamic::translate::make_EIC<inp_control, sender_defs::control_in>("sender1")
 };
 cadmium::dynamic::modeling::EOCs eocs_ABPSimulator = {
-  cadmium::dynamic::translate::make_EOC<Sender_defs::packetSentOut,outp_pack>("sender1"),
-cadmium::dynamic::translate::make_EOC<Sender_defs::ackReceivedOut,outp_ack>("sender1")
+  cadmium::dynamic::translate::make_EOC<sender_defs::packet_sent_out,outp_pack>("sender1"),
+cadmium::dynamic::translate::make_EOC<sender_defs::ack_received_out,outp_ack>("sender1")
 };
 cadmium::dynamic::modeling::ICs ics_ABPSimulator = {
-  cadmium::dynamic::translate::make_IC<Sender_defs::dataOut, inp_1>("sender1","Network"),
-  cadmium::dynamic::translate::make_IC<outp_2, Sender_defs::ackIn>("Network","sender1"),
-  cadmium::dynamic::translate::make_IC<Receiver_defs::out, inp_2>("receiver1","Network"),
-  cadmium::dynamic::translate::make_IC<outp_1, Receiver_defs::in>("Network","receiver1")
+  cadmium::dynamic::translate::make_IC<sender_defs::data_out, inp_1>("sender1","Network"),
+  cadmium::dynamic::translate::make_IC<outp_2, sender_defs::acknowledgement_in>("Network","sender1"),
+  cadmium::dynamic::translate::make_IC<receiver_defs::out, inp_2>("receiver1","Network"),
+  cadmium::dynamic::translate::make_IC<outp_1, receiver_defs::in>("Network","receiver1")
 };
 std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> ABPSimulator = std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(

```

Comment:
Maximum length<=80 characters per line

Reply:
**We did not choose this file for coding conventions**. Please check below files, 

Files assigned to Manoj are:
- message.cpp (src/data_structures),
- main.cpp (test/src/sender),
- sendercadmium.hpp (include/atomics).

Files assigned to Revanth are:
- receivercadmium.hpp (include/atomics),
- subnetcadmium.hpp (include/atomics),
- message.hpp (include/data_structures)